### ISSUE TITLE
AtomMask::InvertMask code change?

### ISSUE THREAD

**Issue by hainm** [Amber-MD/cpptraj#384]
it seems that this 363515c80c1ac7bd754ec9f94d005b93455c66f2 commit broke `pytraj` test.

```python
        atm = AtomMask("@CA")
        assert atm.n_atoms == 0
        top = pt.load_topology("./data/Tc5b.top") # 304 atoms
        top.set_integer_mask(atm)
        assert atm.n_atoms == 20

        top2 = top._modify_state_by_mask(atm)
        assert top2.n_atoms == 20

        atm.invert_mask()
        top3 = top._modify_state_by_mask(atm)
        assert top3.n_atoms == top.n_atoms - 20
```

basically I am expecting to get `top3` with `304 - 20 = 284` atoms. However, I got `275` atoms.

Is this your intention?

**Comment by drroe** [comment]
Nope, bug - I'll fix it.

**Comment by drroe** [comment]
Can you see if it is fixed by #385?

**Comment by hainm** [comment]
yeap.  thanks.
